airflow/models/dag.py

```diff
@@ -2461,7 +2469,7 @@ def create_dagrun(
         :param conf: Dict containing configuration/parameters to pass to the DAG
         :param creating_job_id: id of the job creating this DagRun
         :param session: database session
-        :param dag_hash: Hash of Serialized DAG
+        :param dag_version_id: The DagVersion ID to run with
```

Comment:
```suggestion
        :param dag_version_id: The DagVersion ID for this run
```

Minor thing, but since this is observed, it's probably better to phrase it this way. The bundle version is where we will attempt to hold this stable.